BikeRentalApi/endpoints/stationsBlocked.py

```diff
@@ -0,0 +1,45 @@
+import io
+
+from django.http import JsonResponse
+from rest_framework import status
+from rest_framework.parsers import JSONParser
+
+from BikeRentalApi.enums import Role, StationState
+from BikeRentalApi.models import BikeStation
+from BikeRentalApi.serializers.stationSerializer import StationSerializer
+
+
+def get(user):
+    if user.role != Role.Admin:
+        return JsonResponse({"message": "Forbidden"}, status = status.HTTP_403_FORBIDDEN)
+    stations = BikeStation.objects.filter(state = StationState.Blocked)
+    serializer = StationSerializer(stations, many = True)
+
+    return JsonResponse(serializer.data, safe = False, status = status.HTTP_200_OK)
+
+
+def post(request, user):
+    if user.role != Role.Admin:
+        return JsonResponse({"message": "Forbidden"}, status = status.HTTP_403_FORBIDDEN)
+
+    stream = io.BytesIO(request.body)
+    data = JSONParser().parse(stream)
+
+    try:
+        id = int(data['id'])
+    except KeyError:
+        return JsonResponse({"message": "Bad data"}, status = status.HTTP_400_BAD_REQUEST)
+
+    station = BikeStation.objects.filter(id = id).first()
```

Comment:
For single object better to use `BikeStation.objects.get()` than `filter`

Reply:
done